docs/workflows.md

```diff
@@ -254,6 +254,24 @@ You can use the <a class="tdlp" href="#continue-as-new">Continue-As-New<span cla
 The Workflow Execution spawned from Continue-As-New has the same Workflow Id, a new Run Id, and a fresh Event History and is passed all the appropriate parameters.
 For example, it may be reasonable to use Continue-As-New once per day for a long-running Workflow Execution that is generating a large Event History.
 
+### Size constraints
```

Comment:
Size constraints doesn't feel like the right section title for this.

Maybe just "Limits" ?

We already call out OSS Event hard limit here: https://docs.temporal.io/workflows#event-history

Maybe this limits section could be source of truth and we link to it from Event History ?